[2024-07-03T09:57:13.551+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T06:50:00+00:00 [queued]>
[2024-07-03T09:57:13.566+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T06:50:00+00:00 [queued]>
[2024-07-03T09:57:13.567+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T09:57:13.582+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 06:50:00+00:00
[2024-07-03T09:57:13.589+0000] {standard_task_runner.py:57} INFO - Started process 20488 to run task
[2024-07-03T09:57:13.596+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T06:50:00+00:00', '--job-id', '326', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmpj998e4ww']
[2024-07-03T09:57:13.597+0000] {standard_task_runner.py:85} INFO - Job 326: Subtask id_task_get_emails
[2024-07-03T09:57:13.681+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T06:50:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T09:57:13.827+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T06:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T06:50:00+00:00'
[2024-07-03T09:57:13.828+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T09:57:13.828+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 22, in get_emails
    email_conf = json.load (open(path / ("config/email_conf.json")))
NameError: name 'json' is not defined
[2024-07-03T09:57:13.853+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T065000, start_date=20240703T095713, end_date=20240703T095713
[2024-07-03T09:57:13.877+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 326 for task id_task_get_emails (name 'json' is not defined; 20488)
[2024-07-03T09:57:13.888+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T09:57:13.957+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:04:51.039+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T06:50:00+00:00 [queued]>
[2024-07-03T12:04:51.050+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T06:50:00+00:00 [queued]>
[2024-07-03T12:04:51.050+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:04:51.070+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 06:50:00+00:00
[2024-07-03T12:04:51.077+0000] {standard_task_runner.py:57} INFO - Started process 23309 to run task
[2024-07-03T12:04:51.088+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T06:50:00+00:00', '--job-id', '768', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmpdg64eox_']
[2024-07-03T12:04:51.090+0000] {standard_task_runner.py:85} INFO - Job 768: Subtask id_task_get_emails
[2024-07-03T12:04:51.197+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T06:50:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:04:51.353+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T06:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T06:50:00+00:00'
[2024-07-03T12:04:51.367+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:04:51.369+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:51.369943_User@StatDep: EmailRobot in screated.
[2024-07-03T12:04:51.370+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:51.370474_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:04:51.370+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:51.370855_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:04:51.757+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:04:52.197+0000] {logging_mixin.py:151} INFO -    mailbox have 1 emails and the robot can process 10
[2024-07-03T12:04:52.659+0000] {logging_mixin.py:151} INFO -    #>2024-07-03 12:04:52.659336_conneciton to db is set.
[2024-07-03T12:04:52.678+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 25, in get_emails
    robot.fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 62, in fetch_emails
    return self.pop3_fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 156, in pop3_fetch_emails
    cursor.execute (query, insert_values)
psycopg2.errors.UndefinedTable: relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^

[2024-07-03T12:04:52.703+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T065000, start_date=20240703T120451, end_date=20240703T120452
[2024-07-03T12:04:52.734+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 768 for task id_task_get_emails (relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^
; 23309)
[2024-07-03T12:04:52.771+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T12:04:52.978+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:22:22.378+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T06:50:00+00:00 [queued]>
[2024-07-03T12:22:22.406+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T06:50:00+00:00 [queued]>
[2024-07-03T12:22:22.407+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:22:22.428+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 06:50:00+00:00
[2024-07-03T12:22:22.434+0000] {standard_task_runner.py:57} INFO - Started process 24134 to run task
[2024-07-03T12:22:22.439+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T06:50:00+00:00', '--job-id', '1047', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmp8vtpdquv']
[2024-07-03T12:22:22.440+0000] {standard_task_runner.py:85} INFO - Job 1047: Subtask id_task_get_emails
[2024-07-03T12:22:22.508+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T06:50:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:22:22.647+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T06:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T06:50:00+00:00'
[2024-07-03T12:22:22.647+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:22:22.648+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:22:22.648458_User@StatDep: EmailRobot in screated.
[2024-07-03T12:22:22.648+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:22:22.648642_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:22:22.648+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:22:22.648782_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:22:23.023+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:22:23.335+0000] {logging_mixin.py:151} INFO -    mailbox have 0 emails and the robot can process 10
[2024-07-03T12:22:23.335+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:22:23.335584_User@StatDep: no emails to process, exiting pop3_fetch_emails()...
[2024-07-03T12:22:23.336+0000] {python.py:194} INFO - Done. Returned value was: True
[2024-07-03T12:22:23.368+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T065000, start_date=20240703T122222, end_date=20240703T122223
[2024-07-03T12:22:23.421+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-07-03T12:22:23.455+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
